api/api.go

```diff
@@ -4,11 +4,19 @@ import (
 	"encoding/json"
 	"net/http"
 	"strings"
+	"reflect"
+	"math"
+	"strconv"
 
 	"github.com/NebulousLabs/Sia/build"
 	"github.com/NebulousLabs/Sia/modules"
 )
 
+const (
+	// size of each page for paginated API responses
+	DEFAULT_PAGINATION_SIZE = 25
```

Comment:
We can probably default to something more like 1000. Also, convention is to put constants like this in `consts.go`, and the naming scheme would be `DefaultPaginationSize` instead of using all caps and underscores.